### PR TITLE
add r-wemix

### PR DESCRIPTION
**Pull request by khusmann**
Used https://github.com/bgruening/conda_r_skeleton_helper
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by khusmann** [comment]
looks like it's trying to pull in mro-base_impl on windows... no idea what's going on. I don't see anything about mro in these deps feedstocks. if anyone has a moment to glance at this, it would be appreciated!

```
Package mro-base_impl conflicts for:
r-matrix -> mro-base[version='>=3.5.1,<3.5.2.0a0'] -> mro-base_impl==3.5.1=0
r-minqa -> mro-base[version='>=3.5.1,<3.5.2.0a0'] -> mro-base_impl==3.5.1=0
r-lme4 -> mro-base[version='>=3.5.1,<3.5.2.0a0'] -> mro-base_impl==3.5.1=0
r-numderiv -> mro-base[version='>=3.5.1,<3.5.2.0a0'] -> mro-base_impl==3.5.1=0

Package r-mass conflicts for:
r-npflow -> r-ggplot2 -> r-mass
r-lme4 -> r-mass

...
```

**Comment by xhochy** [comment]
Seems like `r-rmpfr ` is not available in a recent version on Windows.

File: recipes/r-wemix/build.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            install_name_tool -change /usr/lib/libexpat.1.dylib "${PREFIX}"/lib/libexpat.1.dylib "${SHARED_LIB}" || true
+            install_name_tool -change /usr/lib/libcurl.4.dylib "${PREFIX}"/lib/libcurl.4.dylib "${SHARED_LIB}" || true
+            install_name_tool -change /usr/lib/libc++.1.dylib "${PREFIX}"/lib/libc++.1.dylib "${SHARED_LIB}" || true
+            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libc++.1.dylib "${PREFIX}"/lib/libc++.1.dylib "${SHARED_LIB}" || true
+          done <   <(find . \( -type f -iname "*.dylib" -or -iname "*.so" -or -iname "R" \) -print0)
+        popd
+      done
+    popd
+  fi
+fi
```
Comment (xhochy) [inline recipes/r-wemix/build.sh:36]: All this clutter is nowadays not needed anymore
```suggestion
export DISABLE_AUTOBREW=1
${R} CMD INSTALL --build .
```

**Comment by khusmann** [comment]
Thanks @xhochy ! I just tried disabling the windows build, but it doesn't like that as a noarch. Looks like I should jump into the r-rmpfr build for windows then!

Two questions, if you don't mind me asking... (I'm a newbie trying to learn)

1) What part of the error output hinted that it was r-rmpfr?

2) The build.sh clutter came from conda_r_skeleton_helper -- is this not needed for any package anymore? should I make an issue there to not include the boilerplate?

Thanks again!

**Comment by dbast** [comment]
As this is a `noacrch` package, it will build on Linux after feedstock conversion and successfully upload... Thus this PR can be merged... On other platforms (Win, OSX) it will be not installable, if dependencies are missing.

The build.sh clutter is only required in very rare cases for binary repack (to be anyway avoided). A PR against this could remove it as default in the noarch case: https://github.com/conda/conda-build/blob/master/conda_build/skeletons/cran.py

**Comment by dbast** [comment]
@khusmann Thanks

**Pull request merged**